fe1-web/jest/setupAfterEnv.js

```diff
@@ -0,0 +1,24 @@
+import FakeTimers from '@sinonjs/fake-timers';
```

Comment:
Given this file is about fake timers, why not rename it as such? 
setupFakeTimers.js